app/api/api_v1/endpoints/login.py

```diff
@@ -74,8 +117,30 @@ def reset_password(
     new_password: str = Body(...),
     db: Session = Depends(deps.get_db),
 ) -> Any:
-    """
-    Reset password
+    """reset_password: Reset password
+
+    Parameters
+    ----------
+    token : str
+        User token generated before, by default Body(...)
+    new_password : str
+        New user entered password, by default Body(...)
+    db : Session
+        The database session, by default Depends(deps.get_db)
+
+    Returns
+    -------
+    Any
+        Message confirmation for password reset.
+
+    Raises
+    ------
+    HTTPException
```

Comment:
We can aggregate exceptions of same type and just list the different situations such exception might happen. E.g.:

```
HTTPException
    Raised when the user is not found in the system or is inactive.
```